src/azure-cli/azure/cli/command_modules/botservice/custom.py

```diff
@@ -612,8 +611,8 @@ def publish_app(cmd, client, resource_group_name, resource_name, code_dir=None,
     # 2. We shouldn't delete their local web.config and issnode.yml files (if they exist).
     iis_publish_info = {
         'lang': CSHARP if not os.path.exists(os.path.join(code_dir, 'package.json')) else JAVASCRIPT,
-        'has_web_config': True if os.path.exists(os.path.join(code_dir, 'web.config')) else False,
-        'has_iisnode_yml': True if os.path.exists(os.path.join(code_dir, 'iisnode.yml')) else False
+        'has_web_config': bool(os.path.exists(os.path.join(code_dir, 'web.config'))),
```

Comment:
I don't think `bool()` is necessary

Reply:
It was the pylint error to convert these.